src/umap/umap.cpp

```diff
@@ -303,7 +303,7 @@ uint64_t* umap_cfg_readenv(const char* env, uint64_t* val) {
   char* val_ptr = 0;
   if ( (val_ptr = getenv(env)) ) {
     uint64_t env_val = 0;
-    if (sscanf(val_ptr, "%"PRIu64, &env_val)) {
+    if (sscanf(val_ptr, "%" PRIu64, &env_val)) {
```

Comment:
Did you mean to insert a tab? Previously there are lots of lines that take out that character.

Reply:
That's fine then. The space showed up highlighted in darker green in the view and there were lots of prior changes that eliminated the darker green character at the end of the line. I thought it was a tab and you were getting rid of tabs. My misunderstanding.